app/client/components/organizationWrapper.jsx

```diff
@@ -26,7 +26,7 @@ const OrganizationWrapper = Component =>
         }
       } else {
         this.props.history.replace({
-          search: generateQueryString()
+          search: location.search
```

Comment:
This was causing url queries to be wiped if they weren't captured/read by a rendered component before the organisation wrapper wiped the query string.